src/Standards/ISAAC/ruleset.xml

```diff
@@ -126,6 +126,7 @@
             <property name="spacesCountAroundEqualsSign" value="0"/>
         </properties>
     </rule>
+    <rule ref="SlevomatCodingStandard.TypeHints.DisallowArrayTypeHintSyntaxSniff"/>
```

Comment:
I think this should be `SlevomatCodingStandard.TypeHints.DisallowArrayTypeHintSyntax` (with the `Sniff` suffix).

Reply:
Thanks. Resolved issue with an amend.